core/base/src/TApplication.cxx

```diff
@@ -541,8 +541,12 @@ void TApplication::GetOptions(Int_t *argc, char **argv)
                   fFiles->Add(new TNamed("NOT FOUND!", argv[i]));
                   // only warn if we're plain root,
                   // other progs might have their own params
-                  if (!strcmp(gROOT->GetName(), "Rint"))
+                  if (!strcmp(gROOT->GetName(), "Rint")) {
                      Warning("GetOptions", "macro %s not found", fname.Data());
```

Comment:
```suggestion
                     Error("GetOptions", "macro %s not found", fname.Data());
```

Reply:
Thanks @ferdymercury! If we agree that exiting with the error code 2, your suggestion makes a lot of sense - in any case it's too late to recover, and warning seems indeed "weak".